### PR TITLE
WIP: Doc fixes

### PR DESCRIPTION
**Pull request by dpryan79**
These are documentation fixes for Galaxy and read the docs. I'm mostly correcting spelling mistakes and awkward English.

### CODE REVIEW

File: galaxy/wrapper/bamCompare.xml [hunk 1]
```diff
@@ -157,9 +157,9 @@
                 <param argument="--ignoreForNormalization" type="text" value="" size="50"
                     label="regions that should be excluded for calculating the scaling factor"
                     help="Sometimes it makes sense to exclude certain regions when calculating the scaling factor.
-                        For example, if you know some regions that you suspect to be present more often in your sample's genome than in
-                        the reference genome that will therefore accumulate reads (CNV). Another typical example is the single X chromosome
-                        in male samples that should be scaled separately from the diploid autosomes. For example chrX,chrY,chr3. or chr10:12220-128932" />
+                        For example, if you know of copy number variations between samples then you may want to exclude these.
+                        Another typical example is the difference in chromosome X copies between males and females in many species.
+                        Example inputs are chrX,chrY,chr3. or chr10:12220-128932" />
```
Comment (bgruening) [inline galaxy/wrapper/bamCompare.xml]: The dot can be removed, I guess?

**Comment by bgruening** [comment]
Very nice changes!

**Comment by dpryan79** [comment]
@bgruening: Could you run the Galaxy tests on this? Since I updated all of the wrappers I want to make sure I only changed the text and didn't screw anything up :)

**Comment by bgruening** [comment]
Sure!

**Comment by bgruening** [comment]
Works!

**Comment by dpryan79** [comment]
Thanks!

**Comment by dpryan79** [comment]
@bgruening Can you work your magic to get the wrapper updates in the toolshed?

**Comment by bgruening** [comment]
There will be a new Docker Image in one hour!
Thanks a bunch for all the fixes!

**Pull request merged**